oblt-cli/deploy-my-kibana/action.yml

```diff
@@ -0,0 +1,140 @@
+name: 'oblt-cli/deploy-my-kibana'
+description: 'Run the deploy my Kibana PR.'
+inputs:
+  comment-url:
+    description: 'The GitHub Comment URL'
+    default: ${{ github.event.comment.html_url }}
+  comment-id:
+    description: 'The GitHub Comment ID'
+    default: ${{ github.event.comment.id }}
+  issue-url:
+    description: 'The GitHub Issue URL'
+    default: ${{ github.event.comment.issue_url }}
+  repository:
+    description: 'The GitHub repository'
+    default: ${{ github.repository }}
+  user:
+    description: 'The GitHub user that triggered the workflow'
+    default: ${{ github.triggering_actor }}
+  github-token:
+    description: 'The GitHub Personal Access Token.'
+    required: false
+  github-app-id:
+    description: 'The GitHub App ID to generate the ephemeral token.'
+    required: false
+  github-app-private-key:
+    description: 'The GitHub App Private Key to generate the ephemeral token.'
+    required: false
+
+outputs:
+  issue:
+    description: 'The GitHub issue that has been created to destroy the cluster'
+    value: ${{ steps.deploy-my-kibana.outputs.issue }}
+
+runs:
+  using: "composite"
+  steps:
+    - uses: elastic/oblt-actions/github/comment-reaction@v1
+      with:
+        repository: ${{ inputs.repository }}
+        comment-id: ${{ inputs.comment-id }}
+        github-token: ${{ github.token }}
+
+    - if: ${{ (inputs.github-token == '' && inputs.github-app-id == '' && inputs.github-app-private-key == '') || (inputs.github-token != '' && inputs.github-app-id != '' && inputs.github-app-private-key != '') }}
+      name: Validate input parameters
+      run: echo "use either github-token or github-app-id and github-app-private-key" && exit 1
+      shell: bash
+
+    - name: Get token
+      if: ${{ inputs.github-token == '' }}
+      id: get_token
+      uses: tibdex/github-app-token@3beb63f4bd073e61482598c45c71c1019b59b73a # v2.1.0
+      with:
+        app_id: ${{ inputs.github-app-id }}
+        private_key: ${{ inputs.github-app-private-key }}
+        installation_retrieval_mode: organization
+        installation_retrieval_payload: elastic
```

Comment:
See https://github.com/elastic/oblt-actions/pull/144

Reply:
@reakaleek , if you use the GH app action in a GH repository which does not have installed the GH app, then you need these two parameters, otherwise you will see some errors like the ones in https://github.com/elastic/oblt-actions/pull/144

Reply:
Uh, interesting. thanks for the info

Reply:
```
installation_retrieval_mode: id
...
```

does not work, see https://github.com/tibdex/github-app-token/issues/102